core/src/Kokkos_HPX.hpp

```diff
@@ -1269,53 +1422,59 @@ class ParallelScan<FunctorType, Kokkos::RangePolicy<Traits...>,
     thread_buffer &buffer = Kokkos::Experimental::HPX::impl_get_buffer();
     buffer.resize(num_worker_threads, 2 * value_size);
 
+    using hpx::apply;
     using hpx::lcos::local::barrier;
-    using hpx::parallel::for_loop;
-    using hpx::parallel::execution::par;
-    using hpx::parallel::execution::static_chunk_size;
+    using hpx::lcos::local::latch;
 
     barrier bar(num_worker_threads);
+    latch num_tasks_remaining(num_worker_threads);
+    ChunkedRoundRobinExecutor exec(num_worker_threads);
+
+    for (int t = 0; t < num_worker_threads; ++t) {
+      apply(exec, [this, &bar, &buffer, &num_tasks_remaining, num_worker_threads,
+             value_count, value_size, t]() {
```

Comment:
[Jenkins complains](https://cloud.cees.ornl.gov/jenkins-ci/blue/organizations/jenkins/Kokkos/detail/Kokkos/110/pipeline/) about the format here.  Please fix.  (Other error reported makes no sense to me)

Reply:
Done.